core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    {% for element in elements %}
```

Comment:
Is it (semantically) correct that no accordion element is wrapped in a container but just an alternating list of headlines and divs?

Nevertheless, I would introduce a block for that like `{% block element %}` or something, so that you could add one like so:

```twig
{% extends "@Contao/content_element/accordion.html.twig" %}

{# Group elements in sections #}
{% block element %}<section>{{ parent() }}</section>{% endblock %}
```

Reply:
Yes, we use the default markup: https://github.com/oncode/handorgel#markup